src/bgg/norm_simulator.rs

```diff
@@ -132,7 +122,14 @@ impl Evaluable for NormSimulator {
         _: &mut PublicLut<Self::Matrix>,
         _: Option<(Self::Matrix, PathBuf, usize, usize)>,
     ) -> Self {
-        todo!("norm simulator haven't implemented public_lookup")
+        Self {
+            // |c_z · r_k.decompose()| + c_lt_k
+            h_norm: self.h_norm.right_rotate(self.dim_sqrt as u64 * (self.base as u64 - 1)) +
+                MPolyCoeffs::one(),
+            plaintext_norm: BigUint::from(self.base - 1),
```

Comment:
I think the `plaintext_norm` is the maximum norm of the lookup output.

Reply:
Actually as x_k and y_k norm is same, isn't plaintext norm is same as self plaintext norm

Reply:
Here, we don't put any assumption about what function is expressed by a lookup table.
> Actually as x_k and y_k norm is same
So this is not always true in general.
> Also could you explain why did you thought like this
Because the plaintext of the encoding corresponding to the lookup output is a certain output in the lookup table.